src/editpage/components/assistance/SettingsModal.vue

```diff
@@ -174,6 +191,17 @@ export default {
       this.updateConfigurationString();
     },
 
+    exportLogs: function() {
+      const {dialog} = require('electron').remote;
+      const options = {defaultPath: 'logs.zip'};
+      const targetDir = dialog.showSaveDialogSync(options);
+      const electron = require('electron');
```

Comment:
We don't need to import `electron` twice here. You can just reuse the `remote` from above